content/pages/about.md

```diff
@@ -8,7 +8,7 @@ Chennai come together to share their knowledge and to contribute to
 the promotion of Python.
 
 
-<div class="message">NOTE: Upcoming event template, update and uncomment</div>
+<!--<div class="message">NOTE: Upcoming event template, update and uncomment</div>-->
```

Comment:
Don't exactly remember why we have the message class. But the text is supposed to be in comments.